css/mediaQueries.css

```diff
@@ -0,0 +1,30 @@
+/* === media queries === */
+
+@media screen and (max-width: 870px){
```

Comment:
media queries are more often than not, placed at the end of the file. so you can follow up on that

Reply:
yes i have created a separate css file for media queries, and imported that in the end of main.css